lib/auth0/mixins/validation.rb

```diff
@@ -286,7 +286,7 @@ def jwks_url(url, lifetime: 10 * 60)
 
             # Clear the JWK set cache.
             def remove_jwks
-              @@cache.remove(:jwks)
+              @@cache.remove_by { true }
```

Comment:
The cache only exists for RS256 validation, and purging a cache should always be recoverable regardless of what’s in it.  Also, this is only an improvement over the existing implementation, because the cache wasn’t behaving properly when more than one tenant was in play. 

With that said, I could prefix cache-keys with something like `”jwks-“` and only purge keys that start with that. 

If you feel that’s important, I am happy to do that, but I don’t actually see this cache being used for much else.

Reply:
Agree, if that issue happens to occur later, we can solve it then.